static/js/redux/ui/comment_forum.jsx

```diff
@@ -75,14 +156,31 @@ class CommentForum extends React.Component {
       semester_year={this.props.selected_semester.toString().split(' ')[1]}
     />);
 
+      const displayAdvisorNames = () => {
+        const names = []
+        const { advisors, addedAdvisors } = this.state
+        advisors.forEach(({ jhed, name }) => {
+          if (addedAdvisors.includes(jhed)) names.push(name)
+        })
+        return names.join(", ")
+      }
+
     return (
       <div className="comment-forum no-print">
         <div className="cf-name">
-          <p style={{ fontSize: '1.25em', fontWeight: 'bold', marginTop: '70px' }}>
-              Comments Forum
-          </p>
+          <h3 className="title"> Comments Forum</h3>
         </div>
-        <div className="as-header" />
+        {this.props.transcript &&
+        <AdvisorMenu
+            semester_name={semester_name}
```

Comment:
use `selected_semester` here and then parse inside the component :) i believe this will be empty.